app/templates/listen.hbs

```diff
@@ -1,13 +1,26 @@
 <div class="listen">
   <div class="on-air-now">
     <div class="on-air-now-upper">
-    <img class="show-image"
-    alt="image for {{model.stream.currentShow.title}}" src={{model.stream.currentShow.fullImage.url}}>
-    <div class="on-air-now-inner">
-      <div class="on-air-now-label">ON AIR NOW</div>
+    {{#if model.stream.about.people.firstObject.image}}
+      <img class="show-image"
+       alt="image for {{model.stream.currentShow.title}}"
+       src="{{image-template model.stream.about.people.firstObject.image.template 100 100 model.stream.about.people.firstObject.image.crop}}">
+    {{else}}
       <img class="show-image"
        alt="image for {{model.stream.currentShow.title}}"
        src={{model.stream.currentShow.fullImage.url}}>
+    {{/if}}
+    <div class="on-air-now-inner">
+      <div class="on-air-now-label">ON AIR NOW</div>
+      {{#if model.stream.about.people.firstObject.image}}
+        <img class="show-image"
+         alt="image for {{model.stream.currentShow.title}}"
+         src="{{image-template model.stream.about.people.firstObject.image.template 176 176 model.stream.about.people.firstObject.image.crop}}">
+      {{else}}
+        <img class="show-image"
+         alt="image for {{model.stream.currentShow.title}}"
+         src={{model.stream.currentShow.fullImage.url}}>
+      {{/if}}
```

Comment:
Some of this would be good to refactor later—those long string gets could be nicer computed properties on the model, or offloaded to sub-components.